src/resources/HostedInterfacesCore/HostedInterfaceCoreConfig.model.ts

```diff
@@ -169,3 +169,59 @@ export interface HostedInterfaceConfiguration {
      */
     tabs: HostedInterfaceTab[];
 }
+
+export interface IManifestResponse<Configuration> {
+    /**
+     * The HTML markup of the search interface along with configured placeholders.
+     */
+    markup: string;
+    /**
+     * The Atomic results configuration information.
+     */
+    results: IPageManifestResults;
+    /**
+     * The Atomic related styling in CSS format.
+     */
+    style: IPageManifestStyle;
+    /**
+     * The complete search interface configuration.
+     */
+    config: Configuration;
+}
+
+export interface IPageManifestResultTemplate {
+    /**
+     * The HTML markup of the content of the result template.
+     */
+    markup: string;
+    /**
+     * The Atomic [result template component](https://docs.coveo.com/en/atomic/latest/reference/components/atomic-result-template/)'s properties.
```

Comment:
Shouldn't you use the [`@see` tag](https://jsdoc.app/tags-see) to add documentation links?

Reply:
Maybe, but for this PR I'll follow the de facto standard of the repo, which is to do it like this. ;)